src/actions.js

```diff
@@ -0,0 +1,13 @@
+export function updateRegion(region) {
+  return {
+    type: 'updateRegion',
+    payload: {
+      regionName: region,
+    },
+  };
+}
```

Comment:
```suggestion
export function updateRegion(region) {
  return {
    type: 'updateRegion',
    payload: {
      region,
    },
  };
}
```
`regionName`이라는 key 값을 사용하기보다 argument와 같은 `region`로 처리하는건 어떨까요?